hail/python/test/hailtop/utils/test_utils.py

```diff
@@ -126,3 +127,61 @@ def test_parse_docker_image_reference():
     assert x.digest is None
     assert x.name() == 'us-docker.pkg.dev/my-project/my-repo/test-image'
     assert str(x) == 'us-docker.pkg.dev/my-project/my-repo/test-image'
+
+
+
+
+def test_grouped_1():
+    try:
+        actual = list(grouped(0, [1,2,3,4,5,6,7,8,9]))
+        expected = [ 1,2,3,4,5,6,7,8,9]
+    except ValueError:
+        pass
+    else:
+        assert actual == expected
+
+def test_grouped_2():
```

Comment:
Let's make the test function names more descriptive. For example, this test tests that grouping into groups of 1 works, so we might call it: `test_grouped_with_groups_of_one`

Reply:
Changed the names